### DISCUSSION TITLE
python version nailed to exact one version (instead minimum from)

### DISCUSSION THREAD

**Discussion by olmari** [TampereHacklab/mulysa#372] [General]
I there some specific reason we have `python_version = "3.7"` in pipfile (and thus propably similar effects on pipfile.lock? Aka why is it nailed to specific version instead minimum? Otter from Oulu failed to test anything purely becuase python 3.9 is too new ;)

I'm not saying specifically needing to raise minimum supported version, but also not nailing it to specific one if newer available

**Comment by tswfi** [comment]
IIRC pipenv does not support defining different python versions.

Pyenv can be used to install 3.7

Also the docker image will set up the correct python version.

**Comment by olmari** [comment]
Sounds like venv would be more suitable then for this =) Not to say switching wouldn't be otherwise more massive task, but meh... Some clasic case of something is always stupid on something.. Switch tosomethig else and gain new set of issues :)

**Comment by tswfi** [comment]
IIRC I tried to update the python version but your debian version didn't have that yet so we kept this version :D

**Comment by olmari** [comment]
I know, rootissue still is that nailing exact version will always shoot someone in the leg, either the bleeding edge persons can't do stuff or other way stable and LTS persons can't do stuff (easily) :)
